libs/sdk-ui-filters/src/AttributeFIlter@next/AttributeFilter.tsx

```diff
@@ -0,0 +1,326 @@
+// (C) 2007-2022 GoodData Corporation
+import React, { useEffect } from "react";
+import { injectIntl } from "react-intl";
+import MediaQuery from "react-responsive";
+import {
+    filterAttributeElements,
+    IAttributeFilter,
+    isAttributeElementsByRef,
+    isPositiveAttributeFilter,
+    newNegativeAttributeFilter,
+    newPositiveAttributeFilter,
+    ObjRef,
+    IAttributeElement,
+} from "@gooddata/sdk-model";
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+
+import { AttributeDropdown } from "./AttributeDropdown/AttributeDropdown";
+import {
+    AttributeFiltersOrPlaceholders,
+    defaultErrorHandler,
+    IntlTranslationsProvider,
+    IntlWrapper,
+    IPlaceholder,
+    ITranslationsComponentProps,
+    OnError,
+    useCancelablePromise,
+    usePlaceholder,
+    useResolveValueWithPlaceholders,
+    withContexts,
+} from "@gooddata/sdk-ui";
+import { MediaQueries } from "../constants";
+import {
+    attributeElementsToAttributeElementArray,
+    getObjRef,
+    getParentFilterTitles,
+    getValidElementsFilters,
+} from "./utils/AttributeFilterUtils";
+import invariant from "ts-invariant";
+import stringify from "json-stable-stringify";
+
+/**
+ * @public
```

Comment:
again, please mark everything as `@internal`, no public annotation should be in the `AttributeFilter@next` folder for now.

Reply:
fixed